src/data/sampledata.ts

```diff
@@ -1,3 +1,18 @@
+/**
+ * Represents information about an MCAP (Model, Configuration, and Analysis Protocol) file and its associated data.
+ * Each entry in the array corresponds to a specific event, containing metadata about files, content, and event details.
+ *
+ * @typedef {Object} MCAPFileInformationNew
+ * @property {string} id - A unique identifier for the event.
+ * @property {Array<{file_name: string, signed_url: string}>} mcap_files - A list of MCAP files related to the event. Each file is represented by its name and a signed URL to access the file.
+ * @property {Array<{file_name: string, signed_url: string}>} mat_files - A list of MAT files associated with the event. Similar to `mcap_files`, each MAT file has a file name and signed URL.
+ * @property {Array<{content: string, signed_url: string}>} content_files - A list of content files, each having a name (`content`) and a signed URL for access.
```

Comment:
I believe we have a type called ContentFileType, so maybe saying Array<ContentFileType> is better?